docs-source/content/userguide/managing-domains/_index.md

```diff
@@ -33,9 +33,9 @@ Please be aware of the following important considerations for WebLogic domains r
   in the ***Security*** section.
   {{% /notice %}}
 
-* _Log File Locations:_ The operator can automatically override WebLogic domain and server log locations using situational
+* _Log File Locations:_ The operator can automatically override WebLogic domain and server log locations using
   configuration overrides.  This occurs if the domain resource `logHomeEnabled` field is explicitly set to `true`, or if `logHomeEnabled` isn't set
-  and `domainHomeInImage` is explicitly set to `false`.   When overriding, the log location will be the location specified by the `logHome` setting.
+  and `domainHomeSourceType` is set to `PersistentVolume` or `FromModel`.   When overriding, the log location will be the location specified by the `logHome` setting.
```

Comment:
The looks incorrect.  Today, the 'logHome' defaults to overriding iff on PV.   We should continue that behavior as we don't need/want to override unless there's a PV that has the new log file location available. 

So "and `domainHomeSourceType` is set to `PersistentVolume` or `FromModel`. " should be "and `domainHomeSourceType` is set to `PersistentVolume`."   

(Plus make sure the java implementation only changes the defaulting for PV domain source type...)

Reply:
What's the default behavior if domainHomeSourceType is FromModel and without setting logHomeEnabled?  It failed validation if logHomeEnabled is not set.

{
        "timeInMillis" : "1581607100108"
        "body" : ""
        "code" : ""
        "timestamp" : "02-13-2020T15:18:20.108+0000"
        "domainUID" : "domain1"
        "fiber" : "engine-operator-thread-6-fiber-27"
        "message" : "Domain domain1 is not valid: No volume mount contains path for log home '/shared/logs/domain1'"
        "class" : "oracle.kubernetes.operator.helpers.DomainValidationSteps$DomainValidationStep"
        "exception" : ""
        "thread" : "23"
        "level" : "SEVERE"
        "headers" : "{}"
        "method" : "apply"
}

Reply:
Johnny, this validation will go away once I make the change to isLogHomeEnabled() recommended by Tom.